src/components/atoms/ProductsItemList.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from 'react'
+import { useAppDispatch, useAppSelector } from '../../app/hooks'
+import ProductForm from '../molecules/ProductForm'
+import { ProductModal } from '../molecules/ProductModal'
+import { ProductItem } from './ProductItem'
+import { fetchProductsAction } from '../../features/products/actions'
+import { Bounce, ToastContainer } from 'react-toastify'
+import { injectStyle } from 'react-toastify/dist/inject-style'
+
+if (typeof window !== 'undefined') {
+  injectStyle()
+}
```

Comment:
not sure if is a best variant to add styles for toaster, recheck in documentation